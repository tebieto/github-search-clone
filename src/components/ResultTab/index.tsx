import React from 'react';

const ResultTab = (): JSX.Element => {
  return <div></div>;
};

export default ResultTab;
