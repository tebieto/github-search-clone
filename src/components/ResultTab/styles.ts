import styled from 'styled-components';
import colors from '../../utils/colors';

export const ResultTabContainer = styled.div`
  .nav {
    background-color: ${colors.secondaryColor};
    display: inline-block;
    padding: 30px;
    margin: 0;
    border-radius: 5px;
    list-style: none;
    @media screen and (max-width: 800px) {
      display: flex;
      flex-direction: column;
      justify-content: center;
      padding: 0;
      margin: 30px 0;
    }
    .active {
      background-color: ${colors.primaryHoverColor};
    }

    .nav-item {
      padding: 10px;
      min-width: 200px;
      cursor: pointer;
      display: flex;
      justify-content: space-between;
      align-items: center;
      color: ${colors.secondaryTextColor};
      @media screen and (max-width: 800px) {
        min-width: 0;
      }
      .record-count {
        background-color: ${colors.secondaryHoverColor};
        border-radius: 25px;
        font-size: 0.7em;
        padding: 5px 10px;
        color: ${colors.primaryTextColor};
      }
    }
  }
`;
