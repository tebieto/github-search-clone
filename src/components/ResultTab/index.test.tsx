import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ResultTab from '.';

it('renders correctly', () => {
  const props = {
    setActive: () => null,
    active: '',
    users: [],
    repositories: [],
  };
  const tree = shallow(<ResultTab {...props} />);
  expect(toJson(tree)).toMatchSnapshot();
});
