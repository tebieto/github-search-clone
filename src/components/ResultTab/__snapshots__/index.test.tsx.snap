// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<styled.div>
  <ul
    aria-label="sidenav"
    className="nav"
  >
    <li
      aria-label="repository result nav"
      className="nav-item false"
      onClick={[Function]}
    >
      <span>
        Repository
      </span>
       
      <span
        className="record-count"
      >
        0
      </span>
    </li>
    <li
      aria-label="user result nav"
      className="nav-item false"
      onClick={[Function]}
    >
      <span>
        User
      </span>
       
      <span
        className="record-count"
      >
        0
      </span>
    </li>
  </ul>
</styled.div>
`;
